Source/JavaScriptCore/wasm/WasmOperationsInlines.h

```diff
@@ -292,6 +300,8 @@ inline EncodedJSValue arrayNewElem(JSWebAssemblyInstance* instance, uint32_t typ
     JSGlobalObject* globalObject = instance->globalObject();
     VM& vm = globalObject->vm();
     auto* array = JSWebAssemblyArray::create(vm, globalObject->webAssemblyArrayStructure(), FieldType { storageType, Mutability::Mutable }, arraySize, WTFMove(arrayRTT));
+    if (!array)
```

Comment:
UNLIKELY

Reply:
Thanks, wrapping the values in those tests in UNLIKELY() .